taxonomy/management/commands/finalize_xblockskill_tags.py

```diff
@@ -34,49 +34,93 @@ def add_arguments(self, parser):
         Add arguments to the command parser
         """
         parser.add_argument(
-            '--min-votes',
+            '--min-verified-votes',
             help=_('Minimum number of votes required for verification'),
         )
         parser.add_argument(
-            '--ratio-threshold',
-            help=_('Ratio of min votes to total votes for verification'),
+            '--ratio-verified-threshold',
+            help=_('Ratio of min verified_count to total count for verification'),
+        )
+        parser.add_argument(
+            '--min-ignored-votes',
+            help=_('Minimum number of times skill is not verified to be blacklisted'),
+        )
+        parser.add_argument(
+            '--ratio-ignored-threshold',
+            help=_('Ratio of min ignored count to total count for blacklisting'),
         )
 
+    @staticmethod
+    def _get_and_validate_argument_value(argument, setting_key, options):
+        """
+        Gets argument value from arguments or settings. Raises error if None.
+        """
+        value = options.get(argument, None) or getattr(settings, setting_key, None)
+        if value is None:
+            raise InvalidCommandOptionsError(
+                f'Either configure {setting_key} in settings or pass with arg --{argument.replace("_", "-")}'
+            )
```

Comment:
Would this exit the task after raising the error?

Reply:
Yes, raising an exception anywhere will exit the task if not caught. We also have a test for it [here](https://github.com/open-craft/taxonomy-connector/pull/5/files#diff-34a0582d52b032b514b6b6019bda6e5fee67efef7811f2fea388634b9490e123R145-R151).